docs/source/user-guide/integrations/h2o.rst

```diff
@@ -0,0 +1,171 @@
+Using deepchecks with an H2O Model
+==================================
+
+`H2O <https://docs.h2o.ai/h2o/latest-stable/h2o-docs/automl.html>`__ is
+a wildly used AutoML framework, written in Java but with multiple fully
```

Comment:
```suggestion
a widely used AutoML framework, written in Java but with multiple fully
```

Reply:
Personally I think that auto-ML is pretty wild